README.md

```diff
@@ -12,6 +12,15 @@ $~$
 
 $~$
 
+## DISCLAIMER ⚠️
+
+The Starter Templates experience is currently only available to new users.
+
+To benefit from this experience, please follow this link to create a new
+account: [https://www.contentful.com/sign-up/?action=create_starter_template](https://www.contentful.com/sign-up/?action=create_starter_template).
```

Comment:
What do you think of actually adding the template auto installation deep link here? Maybe it was already discussed with Luigi & the team.

Reply:
I'm not sure if it was discussed yet but I'll find out

Reply:
Yeah, it would be a matter of knowing if we want to show all the templates to the users signing up from the GitHub repo or treat them as the Marketing website, with the auto-installation.

Reply:
I thought of replacing the link to keep things simple, but I don't disagree with adding a new paragraph.

Reply:
yeah, I understand but what about users that just want to create a new account and see what the templates are about before installation? Should that not be considered as a different use case? 

I do think it is which is why I added the second link/paragraph so they have both options readily available

Reply:
Yeah I think that is fine 👍